spectroscopy/code_src/mast_functions.py

```diff
@@ -336,6 +340,9 @@ def HST_get_spec(sample_table, search_radius_arcsec, datadir, verbose,
             continue
 
         # Download
+        # Observations.download_products prints an info message for every download, like:
```

Comment:
There should be a kwarg in astroquery.mast to shut these up, if it's missing then please open an issue and we'll get it fixed.

Reply:
I don't see a kwarg, so I'll open an issue.

Reply:
https://github.com/astropy/astroquery/issues/3146

Reply:
Ok, there's a kwarg `verbose=False` that will do this, but we need the latest "dev" version of astroquery (0.4.8.dev9474) to get it. Fornax has 0.4.7 which is the latest version available through pip unless you add the `--pre` flag. @bsipocz have we figured out how to handle these "dev" dependencies in requirements.txt files?
I see `astroquery>=0.4.8.dev0` in the light_curves requirements. Do I remember correctly that if there is any dev version already installed (like on the ISP) it won't be upgraded, but that's still the best option because something like `astroquery>=0.4.8.dev9474` won't work? A slightly older version (0.4.8.dev9306) throws an error, so we'd need more than "dev0" to ensure a working version.

Reply:
I plan to do a release before the end of the year, so for now update your install manually. 
And in the new year astroquery will get a new packaging overhaul and a better system for frequent releases, so no workaround will be needed